rocAL/rocAL_pybind/run.sh

```diff
@@ -23,6 +23,7 @@
 PYTHON_VERSION=`python3 -c "import sys;t='{v[0]}.{v[1]}'.format(v=list(sys.version_info[:2]));sys.stdout.write(t)";` ##Gets the Python3 version
 DEFAULT_PYTHON=$(which python$PYTHON_VERSION) ## Gets the default python
 
+
```

Comment:
Revert this change

Reply:
The run.sh has been removed in the latest TOT after merge with develop